xchange-livecoin/src/main/java/org/knowm/xchange/livecoin/LivecoinAdapters.java

```diff
@@ -258,65 +266,45 @@ public static FundingRecord adaptFundingRecord(Map map) {
         null);
   }
 
-  public static List<Wallet> adaptWallets(List<Map> data) {
-    Map<Currency, WalletBuilder> wallets = new HashMap<>();
-    for (Map balance : data) {
-      String type = balance.get("type").toString();
-      String ccy = balance.get("currency").toString();
-      String value = balance.get("value").toString();
+  public static Wallet adaptWallet(List<LivecoinBalance> livecoinBalances) {
```

Comment:
the previous implementation created a separate wallet for each currency with one balance
changed it to work as the generic API intended - Livecoin have no separate wallets